src/Routing/Asset.php

```diff
@@ -26,17 +26,17 @@
 class Asset
 {
     /**
-     * Inflection type to use when inflection plugin/theme name to get directory name.
+     * Inflection type.
      *
      * @var string
      * @psalm-var 'dasherize'|'underscore'
      */
     protected static $inflectionType = 'underscore';
 
     /**
-     * Set inflection type.
+     * Set inflection type to use when inflecting plugin/theme name.
      *
-     * @param string $inflectionType Inflection type
+     * @param string $inflectionType Inflection type. Valid values: 'dasherize', 'underscore'.
```

Comment:
I recommend making those class const then for clarity and to be easier used by dev.

Reply:
Theoretically one can use any `Inflector` method. Someone can use `'humanize'` if they prefer. I don't want to add constants for all `Inflector` methods. `RouterBuilder` methods also have `inflect` option for similar purpose.

I'll change the description to mention the value should be a valid `Inflector` method name.

Reply:
Cool, ok.